products/centos-stream.md

```diff
@@ -0,0 +1,41 @@
+---
+title: CentOS Stream
+category: os
+iconSlug: centos-stream
+permalink: /centos-stream
+versionCommand: lsb_release --release
+releasePolicyLink: https://wiki.centos.org/About/Product
+releaseLabel: "CentOS Stream __RELEASE_CYCLE__"
+activeSupportColumn: true
+releaseDateColumn: true
+
+identifiers:
```

Comment:
Are those identifiers also valid for CentOS Stream ?

Reply:
I'm not sure where to look to confirm that

Reply:
I looked at the NVD CPE dictionnary : https://nvd.nist.gov/products/cpe. Think we can just replace `centos:centos` by `centos:stream` (I did it, with some miror wording changes).